.github/workflows/ci.yaml

```diff
@@ -0,0 +1,176 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "**"
```

Comment:
The problem is this won't run the workflow for pull requests sent from external contributors which is why I usually opt for master and pull requests instead.

Reply:
My understanding was the CircleCI CI workflow run for all branches, that's why I opted for that approach.

Fixed in b63e66d.